Build failure at `ninja -C build/`

Greetings.
Trying to compile gamescope at ba5f9e8e0c490ebadc1012ff1e6f5a38cb4a00f7 commit without success. 
```shell
ninja -C build/
ninja: Entering directory `build/'
[1/2] Compiling C++ object src/gamescope.p/vr_session.cpp.o
FAILED: src/gamescope.p/vr_session.cpp.o 
c++ -Isrc/gamescope.p -Isrc -I../src -I../src/reshade/source -I../src/reshade/include -I../thirdparty/SPIRV-Headers/include/spirv/unified1 -Isubprojects/wlroots/include -I../subprojects/wlroots/include -I../subprojects/libliftoff/include -Isubprojects/stb -I../subprojects/stb -Iprotocol -I/usr/include -I/usr/include/libdrm -I/usr/include/SDL2 -I/usr/include/pixman-1 -I/usr/include/uuid -I/usr/include/pipewire-0.3 -I/usr/include/spa-0.2 -I/usr/include/openvr -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -Wextra -std=c++20 -O0 -g -DWLR_USE_UNSTABLE -Wno-unused-parameter -Wno-missing-field-initializers -Wno-invalid-offsetof -Wno-unused-const-variable -Wno-volatile -Wno-ignored-qualifiers -Wno-missing-braces -ffast-math -DHAVE_PIPEWIRE=1 -DHAVE_OPENVR=1 '-DHWDATA_PNP_IDS="//usr/share/hwdata/pnp.ids"' -DAVIF_DLL -isystem/usr/include/libdrm -D_REENTRANT -pthread -DHAVE_DRM=1 -DHAVE_SDL2=1 -DHAVE_AVIF=1 -DHAVE_LIBCAP=1 -MD -MQ src/gamescope.p/vr_session.cpp.o -MF src/gamescope.p/vr_session.cpp.o.d -o src/gamescope.p/vr_session.cpp.o -c ../src/vr_session.cpp
../src/vr_session.cpp: In member function <<virtual bool gamescope::COpenVRBackend::Init()>>:
../src/vr_session.cpp:348:62: oshibka: <<VROverlayFlags_EnableControlBar>> is not a member of <<vr>>; did you mean <<VROverlayFlags_EnableControlBarSteamUI>>?
  348 |             vr::VROverlay()->SetOverlayFlag( m_hOverlay, vr::VROverlayFlags_EnableControlBar,                   m_bEnableControlBar );
      |                                                              ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                                                              VROverlayFlags_EnableControlBarSteamUI
../src/vr_session.cpp:349:62: oshibka: <<VROverlayFlags_EnableControlBarKeyboard>> is not a member of <<vr>>; did you mean <<VROverlayFlags_EnableControlBarSteamUI>>?
  349 |             vr::VROverlay()->SetOverlayFlag( m_hOverlay, vr::VROverlayFlags_EnableControlBarKeyboard,   m_bEnableControlBarKeyboard );
      |                                                              ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                                                              VROverlayFlags_EnableControlBarSteamUI
../src/vr_session.cpp:350:62: oshibka: <<VROverlayFlags_EnableControlBarClose>> is not a member of <<vr>>; did you mean <<VROverlayFlags_EnableControlBarSteamUI>>?
  350 |             vr::VROverlay()->SetOverlayFlag( m_hOverlay, vr::VROverlayFlags_EnableControlBarClose,      m_bEnableControlBarClose );
      |                                                              ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
      |                                                              VROverlayFlags_EnableControlBarSteamUI
ninja: build stopped: subcommand failed.
``` 
OS Arch linux.
Tried aur package and manually.
Update: removing openvr arch package resolve this issue.

--- a/meson.build
+++ b/meson.build
@@ -45,7 +45,7 @@ dep_wayland = dependency('wayland-client')
 vulkan_dep = dependency('vulkan')
 
 if get_option('enable_openvr_support')
-  openvr_dep = dependency('openvr', required : false)
+  openvr_dep = dependency('openvr', version: '>= 2', required : false)
   if not openvr_dep.found()
     cmake = import('cmake')
     openvr_var = cmake.subproject_options()

